### DISCUSSION TITLE
What is the best way to authenticate a Webview?

### DISCUSSION THREAD

**Discussion by ManuLpz4** [supabase/supabase#29443] [Questions]
Hey, guys! How is it going?

I'm having headaches trying to implement the following auth flow in my Next.js + NextAuth + Supabase:

I made an SDK that is just a webview that has an accessToken and loads a Next.js page. In the web I have NextAuth and I want to authenticate the user with the token to automatically authenticate every request with Supabase within it.

What is the best way?

Sorry for the long explanation and thank you for reading :)

**Comment by hitesh-c** [comment]
You need to make sure that your next app has access to local storage and cache in your webview.
